src/v/config/rjson_serialization.h

```diff
@@ -15,14 +15,29 @@
 #include "config/data_directory_path.h"
 #include "config/endpoint_tls_config.h"
 #include "config/seed_server.h"
-#include "config/tests/custom_aggregate.h"
 #include "config/tls_config.h"
 #include "json/json.h"
 #include "json/stringbuffer.h"
 #include "json/writer.h"
 
 #include <seastar/core/sstring.hh>
 
+/*
```

Comment:
nit: is it worth wrapping this under a `testing` namespace?

Reply:
yeh good idea. fixed